src/ProtectedRoute.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Navigate } from 'react-router-dom';
+
+interface SomeType {
+  children: any,
+  redirectTo: any
+}
+// eslint-disable-next-line react/prop-types
+export default function ProtectedRoutes(obj: SomeType) {
```

Comment:
On the login page, we should not redirect the people.

You give the people a notice that they are already logged and and they have to logout before changing the account

Otherwise people will be confused

Reply:
Thanks @feyton , can we notice the user that is already logged in and still have access to login page ?